supported-architectures.txt

```diff
@@ -0,0 +1,8 @@
+"386_linux_"
+"amd64_linux_"
+"amd64_windows_10.0.17763.5458"
+"amd64_windows_10.0.20348.2322"
```

Comment:
Does it make sense to pin the actual Windows version here? This version will change; for example, `ltsc2022` is now `10.0.20348.2340` already.

Reply:
You're right, this won't be stable enough. I changed the check to only count the architectures and to not check the actual versions.